scrambles/src/main/java/net/gnehzr/tnoodle/puzzle/CubePuzzle.java

```diff
@@ -50,7 +45,7 @@ public CubeMove(Face face, int dir, int innerSlice, int outerSlice) {
             this.innerSlice = innerSlice;
             this.outerSlice = outerSlice;
             // We haven't come up with names for moves where outerSlice != 0
-            azzert(outerSlice == 0);
+            assert outerSlice == 0;
```

Comment:
Why the change here? IIRC, the reason we use `azzert` instead of `assert` is because we wanted to guarantee that assertions were enabled at runtime, and that wasn't possible to do because it's a parameter the person running TNoodle would need to tweak.

Reply:
I was generally trying to get rid of GWTSafeUtils as part of the restructure.
Using a "forced" assert irrespective of the user settings never actually occured to me, but are you sure that enforcing the hard-wired fails is really necessary? In other words, does the assertion ever actually fail in production?

Reply:
It's hard to know if they hit in production, because we'd only know if people tell us. I don't now of any recent situations in which that has happened. I'm sure it happened in the earlier days.

My thinking is: the code was written with these invariants in mind, and it's really hard to predict the behavior of the code if it keeps running past any of these assertions. It would be really bad an assertion failing to blow caused us to generate a scramble that doesn't obey the WCA regulations for some reason.